src/plugins/intel_gpu/src/kernel_selector/cl_kernels/reorder_data.cl

```diff
@@ -27,8 +27,15 @@ KERNEL (reorder_data)(
 #endif
     )
 {
+#if INPUT0_LAYOUT_BYFX
+    // GWS_FEATURE takes Y for byfx format
+    const uint b = get_global_id(GWS_BATCH);
+    const uint y = get_global_id(GWS_FEATURE);
+#else
     const uint b = get_global_id(GWS_BATCH);
     const uint f = get_global_id(GWS_FEATURE);
+#endif
+
```

Comment:
we have various planar formats, but why do we need special handling for byfx?

Reply:
Because gws is calculated based on b/y/fx.  So here 'f' is replaced by 'y'.